notebooks/community/model_evaluation/automl_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,1473 @@
+{
```

Comment:
<p>This gives me the following error:</p><pre class="ql-syntax" spellcheck="false">AttributeError: 'NoneType' object has no attribute 'artifacts'
</pre><p>Wondering if you could debug?</p><p>Apparently <code>task.outputs.get('feature_attributions')</code>returns <code>None</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>

Reply:
<p>Cross checked again, working fine for me.</p><p> </p>